terra/assetlist.json

```diff
@@ -5732,6 +5732,33 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/trit.png"
         }
       ]
+    },
+    {
+      "description": "Rakoff Token is the Meme Token for Terra Classic",
+      "type_asset": "cw20",
+      "address": "terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+      "denom_units": [
+        {
+          "denom": "cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+          "exponent": 0
+        },
+        {
+          "denom": "rakoff",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:terra1vhgq25vwuhdhn9xjll0rhl2s67jzw78a4g2t78y5kz89q9lsdskq2pxcj2",
+      "name": "Rakoff Token",
+      "display": "Rakoff",
```

Comment:
this unit is not defined. Change to 'rakoff'.

Reply:
Done.